dialogue-core/src/main/java/com/palantir/dialogue/core/BlacklistingChannel.java

```diff
@@ -96,29 +98,27 @@
     }
 
     @Override
-    public Optional<ListenableFuture<Response>> maybeExecute(Endpoint endpoint, Request request) {
+    public LimitedResponse maybeExecute(Endpoint endpoint, Request request) {
         BlacklistState state = channelBlacklistState.get();
         if (state != null) {
             BlacklistStage stage = state.maybeProgressAndGet();
             if (stage instanceof BlacklistUntil) {
-                return Optional.empty();
+                return LimitedResponses.blacklisted();
```

Comment:
Rather than `LimitedResponses.blacklisted`, what do you think of `LimitedResponses.backOff` ?